### DISCUSSION TITLE
物料里面的地址能改成配置吗？

### DISCUSSION THREAD

**Discussion by w990752800** [opentiny/tiny-engine#424] [Q&A]
![image](https://github.com/opentiny/tiny-engine/assets/23731304/19a1bb2e-7f04-4ef3-9064-beed32adcffd)

**Comment by w990752800** [comment]
现在我每加一个都要配置这个域名端口，这个能提取成配置项吗？

**Reply by wenmine** [reply]
当前没有提供这个能力，可以参与共建把你的想法实现起来呀
